pkg/platform/runtime/buildplan/buildplan.go

```diff
@@ -45,6 +47,17 @@ func NewMapFromBuildPlan(build *model.Build) (artifact.Map, error) {
 		}
 	}
 
+	// Eliminate noop artifacts
+	for _, prebuiltArtf := range build.Artifacts {
+		if prebuiltArtf.NodeID != "" && prebuiltArtf.Status != "" &&
+			prebuiltArtf.Status == model.ArtifactSucceeded &&
+			strings.HasPrefix(prebuiltArtf.URL, "s3://as-builds/noop/") {
+			if _, exists := res[prebuiltArtf.NodeID]; exists {
+				delete(res, prebuiltArtf.NodeID)
+			}
+		}
+	}
```

Comment:
Previously we were only doing this for download/install artifacts. But it made sense to extend this to all. There's no point in looking at these at all.

Reply:
Artifact URLs from the buildplanner should not have an `s3` scheme, they should all be `https` so I don't think this will have an effect.